composer.json

```diff
@@ -24,7 +24,7 @@
         }
     },
     "require": {
-        "php": ">=5.6",
+        "php": ">=7.0",
```

Comment:
This probably isn't a good idea since Valet itself still supports PHP 5.6 as a courtesy to the many developers required to support older environments. 

That said, you could update `.travis.yml` to add `7.2` since `nightly` actually builds PHP 7.3 currently.

Reply:
I only dropped support was because the tests were falling. What do you suggest me to do?

Reply:
In `composer.json` use:

```json
        "phpunit/phpunit" : "^5.7|~6.2"
```

Reply:
@drbyte I didn't know I can require one `or (I)` other version. I'll work on that!